framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts

```diff
@@ -1,3 +1,12 @@
+import {
+	ActionsDefinition,
+	BasePlugin,
+	BaseChannel,
+	EventsDefinition,
+	PluginInfo,
+	PluginOptionsWithAppConfig,
+} from 'lisk-framework';
+import { SchemaWithDefault } from '../../../../elements/lisk-chain/src/utils/account';
```

Comment:
this should be chain library import? and we dont need chain we can use
import Schema from codec
`export type SchemaWithDefault = Schema & { default: Record<string, unknown> };`

Reply:
`SchemaWithDefault` is defined in `framework`. Will see why my IDE is getting over intelligent to import from other locations.